<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupFilenamesWindow</class>
 <widget class="QDialog" name="SetupFilenamesWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>769</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Setup Filenames</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>730</y>
     <width>371</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="volumeFilenameTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>391</width>
     <height>171</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>2</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Modality name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Filename</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter filenames relative to subject directory.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Model filename:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="modelFilenameLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nifti files for different modalities:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>500</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton">
   <property name="geometry">
    <rect>
     <x>303</x>
     <y>500</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Filename for adjustments file (output):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="outputFilenameLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="displacementsBasenameLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Base filename for output from mmsubseg:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="transformationFilenameLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Affine or nonlinear transformation to standard space:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>650</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Extra shape to be displayed (optional, NOT transformed):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="extraShapeFilenameLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>670</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="configurationComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::NoInsert</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select configuration (current config is saved when clicking OK)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Exclusion mask for normalisation:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="normExclusionFilenameLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Filenames for multiple shape viewer (uses volumes on left):</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="multiShapeTable">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>501</width>
     <height>281</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>100</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>4</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Filename</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Displacements</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Red</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Green</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Blue</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="multiShapeAddButton">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>310</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multiShapeRemoveButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>311</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="overlayFilenameLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>730</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>710</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mask to be overlaid:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>configurationComboBox</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>modelFilenameLineEdit</tabstop>
  <tabstop>transformationFilenameLineEdit</tabstop>
  <tabstop>displacementsBasenameLineEdit</tabstop>
  <tabstop>volumeFilenameTable</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>outputFilenameLineEdit</tabstop>
  <tabstop>extraShapeFilenameLineEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SetupFilenamesWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SetupFilenamesWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>635</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
